src/main/java/com/openlattice/authorization/EdmAuthorizationHelper.java

```diff
@@ -33,13 +32,18 @@
 import org.springframework.stereotype.Component;
 
 import javax.validation.constraints.NotNull;
-import java.util.*;
+import java.util.EnumMap;
+import java.util.EnumSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
 @Component
 public class EdmAuthorizationHelper implements AuthorizingComponent {
-    public static final EnumSet<Permission>  READ_PERMISSION  = EnumSet.of( Permission.READ );
+    public static final EnumSet<Permission>  TRANSPORT_PERMISSION = EnumSet.of( Permission.MATERIALIZE );
```

Comment:
it would be a pretty simple migration to just rename this permission to TRANSPORT in the permissions table (especially since we need to run migrations anyway)

Reply:
(although then we'd also need something to automatically transform MATERIALIZE to TRANSFORM from inputs coming from API calls, which feels pretty icky so this is probably not worth on second thought)

Reply:
ehh yeah lets just keep it MATERIALIZE and maybe change it later